cmd/address_gen/address_gen.go

```diff
@@ -24,6 +25,8 @@ func main() {
 	genCount := flag.Int("n", 1, "Number of addresses to generate")
 	hideSecKey := flag.Bool("s", false, "Hide the secret key from the output")
 	isBitcoin := flag.Bool("b", false, "Print address as a bitcoin address")
+	hexSeed := flag.Bool("x", false, "Use hex string as seed")
```

Comment:
What is hex string seed for

Reply:
the generator use random hex string as default seed before, we just keep it here.